irohad/main/application.cpp

```diff
@@ -149,6 +149,16 @@ Irohad::RunResult Irohad::init() {
     }
   };
 
+  validators_config_ =
```

Comment:
Please initialize them in constructor as it is done, otherwise some components might get other configs. These assignments reassign the values,

Reply:
Fixed